app/views/_nav_bar.html.erb

```diff
@@ -26,16 +26,10 @@
         <li class="nav-item">
           <%= link_to 'Users', admin_users_path, :class => "nav-link" %>
         </li>
-        <li class="nav-item">
-          <%= link_to "Browse Merchants as Admin", admin_merchants_path, :class => "nav-link"  %>
-        </li>
       <% else current_user && current_user.role == "admin" %>
         <li class="nav-item">
           <%= link_to "Shopping Cart (#{ @cart.items.count })", cart_path, :class => "nav-link"  %>
         </li>
-        <li class="nav-item">
```

Comment:
Since all users need to browse merchants, do you think this could go in the main nav with an if-statement for the path, if the paths need to be different, or if the paths are the same, just an if-statement inside that page for the enable/disable functionality?  I'm sure you've considered; just confirming the plan.  Looks great!